libs/app-serialport/devices/src/lib/serial-port-device.ts

```diff
@@ -118,7 +118,7 @@ export class SerialPortDevice extends SerialPort {
   }
 
   private createRequest({ options, ...data }: SerialPortRequest) {
-    const id = parseInt(uniqueId())
+    const id = parseInt(uniqueId()) % 99999999 || 1
```

Comment:
⚪ 
I understand this line is meant to constrain the ID range. I suggest extracting the ranged ID generation into a dedicated helper or using a single, proven library (e.g. nanoid) so the entire project relies on one consistent method.